packages/components/src/Combobox/components/ComboboxContent/ComboboxContent.tsx

```diff
@@ -11,19 +11,29 @@ interface ComboboxContentProps {
   readonly options: string[];
 
   /**
-   * onSelection handler.
+   * onSelect handler.
    */
-  readonly onSelection: (selection: string) => void;
+  readonly onSelect: (selection: string) => void;
```

Comment:
nit: mostly because it follows the prop name that it eventually fills out

```suggestion
  readonly onSelect: (selected: string) => void;
```